poc/tests/test_vidpf.py

```diff
@@ -35,16 +35,16 @@ def test_eval_invariants(self):
             # The control bits should be secret shares of one, i.e., the
             # control bit should be set for one and only one of the
             # aggregators.
-            self.assertEqual(node[0].ctrl + node[1].ctrl, Field2(1))
+            self.assertTrue(node[0].ctrl ^ node[1].ctrl)
```

Comment:
maybe `node[0].ctrl != node[1].ctrl` for clarity?

Reply:
Done.